panopticum/serializers.py

```diff
@@ -246,6 +256,14 @@ class ComponentVersionSerializer(QueryFieldsMixin, ComponentVersionSerializerSim
     negative_status_count = serializers.IntegerField(read_only=True)
     unknown_status_count = serializers.IntegerField(read_only=True)
 
+    dependent = serializers.SerializerMethodField()
```

Comment:
```dependent = ComponentVersionSerializer(many=True, field='depends_on')``` does not work?

Reply:
No, getting exception like:
TypeError: __init__() got an unexpected keyword argument 'field'

I think you can optimize after commit, ok?